Modules/Forum/classes/class.ilForumAppEventListener.php

```diff
@@ -43,25 +43,6 @@ public static function handleEvent(string $a_component, string $a_event, array $
         switch ($a_component) {
             case 'Modules/Forum':
                 switch ($a_event) {
-                    case 'mergedThreads':
```

Comment:
This `case` MUST NOT be deleted IMO. We need to refresh the learning progress if two threads are combined into one thread.